api/v1alpha1/registry_types.go

```diff
@@ -13,7 +13,9 @@ type RegistrySpec struct {
 	Repositories []string `json:"repositories,omitempty"`
 	// AuthSecret is the name of the secret in the same namespace that contains the credentials to access the registry.
 	AuthSecret string `json:"authSecret,omitempty"`
-	// ScanInterval is the interval at which the registry is scanned. Set to 0 to disable automatic scanning.
+	// ScanInterval is the interval at which the registry is scanned.
+	// If not set, automatic scanning is disabled.
+	// +kubebuilder:validation:Optional
 	ScanInterval metav1.Duration `json:"scanInterval"`
```

Comment:
Shouldn't we define it as a pointer instead?

```golang
ScanInterval *metav1.Duration `json:"scanInterval"`
```

So that will not be shown when we do `kubectl get registry` if it's not set.

Reply:
This is commonly used without a pointer in k8s: 
https://github.com/kubernetes/kubernetes/blob/cb163eb55b83ac54e6af10d9c8b7faaa36d4b141/pkg/proxy/apis/config/types.go#L74